### PR TITLE
Add AD via LDAP provider

### PR DESCRIPTION
**Pull request by pbrezina**

### CODE REVIEW

File: src/docs/ldap-ad.rst [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+This describes how to configure SSSD to setup an Active Directory domain using
+``id_provider = ldap``.
+
+.. note::
+
+    The recommended way to join into an Active Directory domain is to use the
+    integrated AD provider (``id_provider = ad``). See :doc:`direct-ad` for
+    more information.
+
+    The only reason why to use the ``ldap`` provider is if you do not want to
```
Comment (justin-stephenson) [inline src/docs/ldap-ad.rst:13]: s/why//

File: src/docs/ldap-ad.rst [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+            log file = /var/log/samba/%m.log
+            password server = AD.EXAMPLE.COM
+            realm = EXAMPLE.COM
+            security = ads
+
+#.  Join the machine to the realm
+
+    .. code-block: bash
+
+        net ads join -U Administrator
```
Comment (justin-stephenson) [inline src/docs/ldap-ad.rst:106]: This is not visible on the page.

**Comment by justin-stephenson** [comment]
Just the 2 minor changes here, otherwise Ack.

**Comment by pbrezina** [comment]
Fixed. See new patch.

**Comment by justin-stephenson** [comment]
Ack.

**Pull request merged**